src/System.Resources.ResourceManager/ref/project.json

```diff
@@ -1,14 +1,11 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.Reflection": "4.0.0",
-    "System.Globalization": "4.0.0"
+    "System.IO.UnmanagedMemoryStream": "4.3.0-beta-24604-02",
```

Comment:
We may want to push UnmanagedMemoryStream down to System.Runtime.  I think NuGet will still call this a cycle if someone references the old UnmanagedMemoryStream indirectly and the new ResourceManager.